data/avro-schema/src/main/resources/avro/net/corda/data/flow/FlowTimeout.avsc

```diff
@@ -0,0 +1,20 @@
+{
+  "type": "record",
+  "name": "FlowTimeout",
+  "namespace": "net.corda.data.flow",
+  "doc": "Represents of a flow timeout instruction.",
+  "fields": [
+      {
+        "name": "checkpointStateKey",
+        "type": "string",
+        "doc": "Key for the state record that is storing the checkpoint."
+      },
+      {
+        "name": "timeoutDateTime",
```

Comment:
This is probably not strictly necessary but may be useful when troubleshooting or monitoring the system health?